internal/serialize/keyvalues.go

```diff
@@ -93,6 +93,51 @@ func TrimDuplicates(kvLists ...[]interface{}) [][]interface{} {
 	return outs
```

Comment:
(wanted to comment on TrimDuplicates function, but github...)
Are we still using `TrimDuplicates` anywhere in code? If it's not needed for klog should we plan it's removal by marking it as deprecated?

Reply:
We are not using it, but because it is part of the klog/v2 API we cannot remove it.

I was thinking about marking it as deprecated, but then decided against it because there might be users of it who need the additional functionality (merging more than two slices) and causing an error message for them that they cannot get rid of easily felt wrong. But this is up for debate.

Reply:
I would be a little cautious about trying to provide stable utils library as part of klog package. We should focus on making a logging library and not a generic tooling for metadata processing. We should avoid trying to achieve to many things.

This is however a larger discussion about goals and charter for klog.

Reply:
Oh, wait!

/hold

I intentionally put this function into "internal".

Reply:
Which means we *can* remove it. Let me try that in a separate PR.